data/sql/updates/pending_db_world/rev_1609109042026647800.sql

```diff
@@ -0,0 +1,367 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1609109042026647800');
+
+-- ----------------------------
+-- Table structure for gtoctclasscombatratingscalar_dbc
+-- ----------------------------
+DROP TABLE IF EXISTS `gtoctclasscombatratingscalar_dbc`;
+CREATE TABLE `gtoctclasscombatratingscalar_dbc`  (
+  `ID` int(11) NOT NULL DEFAULT 0,
+  `Data` float NOT NULL DEFAULT 0,
+  PRIMARY KEY (`ID`) USING BTREE
+) ENGINE = MyISAM CHARACTER SET = utf8 COLLATE = utf8_general_ci ROW_FORMAT = Fixed;
+
+-- ----------------------------
+-- Records of gtoctclasscombatratingscalar_dbc
+-- ----------------------------
+INSERT INTO `gtoctclasscombatratingscalar_dbc` VALUES (1, 1);
```

Comment:
why so many `INSERT INTO` when you can just use one as explained here: https://www.azerothcore.org/wiki/Dealing-with-SQL-files#write-compact-code

Reply:
okay